src/lib/REST/Value/SubItems/ContentInfo.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\REST\Value\SubItems;
+
+use Ibexa\Rest\Value;
+
+final class ContentInfo extends Value
+{
+    public function __construct(
+        public int $id,
```

Comment:
Can properties or the whole class be `readonly`?

Reply:
Class no, due to extanding Value, but :+1: for properties.